Formula/xapian.rb

```diff
@@ -39,6 +28,9 @@ class Xapian < Formula
   deprecated_option "ruby" => "with-ruby"
 
   depends_on :python => :optional
+  depends_on "sphinx-doc" if build.with? "python" => :build
```

Comment:
This is ordered slightly wrong. You want:
```ruby
depends_on "sphinx-doc"  => :build if build.with? "python"
```
Is that a hard requirement though? Is there no documentation unless you build with bindings?